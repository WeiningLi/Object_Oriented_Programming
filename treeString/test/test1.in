ra "Hello cs246"
^a6 "the greatest (?) "
pa
rb ", how are you?"
pb
cdb
+cad
pc
ia1?
pa
da
pa
q
