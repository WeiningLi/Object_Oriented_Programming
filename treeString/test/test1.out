Hello the greatest (?) cs246
, how are you?
Hello the greatest (?) cs246, how are you?
H?llo the greatest (?) cs246
"the greatest (?) " size: 23
 "H?llo " size: 6
 "cs246" size: 5
H?llo the greatest (?) cs246
